### ISSUE TITLE
Constant occupancy check giving false positives

### ISSUE THREAD

**Issue by ian-r-rose** [cagov/caldata-mdsa-caltrans-pems#231]
I was looking at some of the station statuses, and noticed that the constant occupancy check seems to be giving some false positives.

For some stations it looks correct. Here is station id `774327`, lane 1 on 2023-11-02:

![visualization(1)](https://github.com/cagov/caldata-mdsa-caltrans-pems/assets/5728311/60952dbd-2360-4b15-988a-5453582ed2af)

For others it does not look correct. Here is station id `1208809`, lane 3, on 2023-04-07:

![visualization(2)](https://github.com/cagov/caldata-mdsa-caltrans-pems/assets/5728311/e27c53f0-3f75-4962-af99-7b8496535593)

I'm not sure what the issue is, though I suspect that the `preceding` window function might not be handling the start-of-day appropriately.

Code used to produce the above charts (without imports and the connection creation):

```python
r = conn.cursor().execute(
    "select * from transform_prd.clearinghouse.int_clearinghouse__five_minute_station_agg"
    " where id = '1208809' and sample_date = '2023-04-07' and lane = '3'"
)
df = r.fetch_pandas_all()

altair.Chart(df[["SAMPLE_TIMESTAMP", "OCCUPANCY_AVG"]]).mark_line().encode(
    x="SAMPLE_TIMESTAMP:T",
    y="OCCUPANCY_AVG:Q",
).interactive()
```

**Comment by britt-allen** [comment]
Is this issue related to [this](https://california.getdbt.com/deploy/6/projects/10/runs/3254/)?

**Comment by ian-r-rose** [comment]
No, that issue is being tracked in #228

**Comment by britt-allen** [comment]
I had too many tabs open. I meant to post that question in issue 228, not here! Thanks!

**Comment by ian-r-rose** [comment]
Fixed by #248
